### PR TITLE
Added module-level docstring showing basic use

### PR DESCRIPTION
**Pull request by WeatherGod**
Don't know if I got my REST syntax right, though..., but it now shows simple examples when one does "pydoc cycler".

### CODE REVIEW

File: cycler.py [hunk 1]
```diff
@@ -1,3 +1,40 @@
+"""
+Cycler
+======
+
+Cycling through combinations of values, producing dictionaries.
+
+You can add cyclers::
```
Comment (tacaswell) [inline cycler.py]: You need a blank line between these sections for the parsing to work right.

Comment (WeatherGod) [inline cycler.py]: Done

**Comment by WeatherGod** [comment]
Gah, I forgot to set my clone's email settings. Does anybody care enough about that?

**Comment by tacaswell** [comment]
We can clean that up with a `.mailmap` file.

**Pull request merged**